src/index.ts

```diff
@@ -1,21 +1,93 @@
+
+import {
+    INotebookTracker
+} from '@jupyterlab/notebook';
+
+import {
+  ContentsManager
+} from '@jupyterlab/services';
+
+import {
+  ICommandPalette
+} from '@jupyterlab/apputils';
+
 import {
   JupyterLab,
   JupyterLabPlugin
 } from '@jupyterlab/application';
 
+import {AllOfUsConfig} from './config';
+import {ConceptsWidget} from './concepts-widget';
+
 import '../style/index.css';
 
+// Activate the jupyterhub extension.
+function activateExtension(app: JupyterLab,
+                           notebooks: INotebookTracker,
+                           palette: ICommandPalette): void {
+    const tracker = notebooks;
```

Comment:
Why? Can you just name the parameter differently? Or just not rename it?

Reply:
Switched to not rename this.